tests/installation/releasenotes_offline.pm

```diff
@@ -21,11 +21,12 @@ sub run {
     assert_screen('release-notes-button');
     send_key('ctrl-shift-alt-x');
     assert_screen('yast-xterm');
-    my $src = check_var('SCC_REGISTER', 'installation') ? "RPM" : "URL";
+    # This is releasenotes_offline so we expect release notes from RPM
+    my $src = "RPM";
```

Comment:
No need for variable, we just need to put the value in `match_has_tag` and `record_soft_failure`.

Just a hint, but not everybody agree, if you don't have interpolation of strings, just use single quotes (some people prefers to use double so in the future if is expected interpolation, they are already there, but in general I've seen that not much useful).